nncf/openvino/graph/transformations/command_creation.py

```diff
@@ -54,3 +57,11 @@ def create_command_to_update_weight(
     def create_command_to_insert_bias(node_without_bias: NNCFNode) -> OVNullBiasInsertionCommand:
         target_point = OVTargetPoint(TargetType.POST_LAYER_OPERATION, node_without_bias.node_name, 0)
         return OVNullBiasInsertionCommand(target_point)
+
+    @staticmethod
+    def multiply_insertion_command(
```

Comment:
Should it be called `multiplies_insertion_command` as it could insert several multiplies at onece?